src/three-components/envmap_physical_pars_fragment.glsl.js

```diff
@@ -0,0 +1,135 @@
+export default /* glsl */ `
```

Comment:
There seems to be some pretty inconsistent indentation here. Is it possible to align it with `cube_uv_reflection_fragment`?

Reply:
I left it intentionally with unchanged formatting from the three.js version since I only changed two lines and this makes the diff a lot nicer.